docs/pages/eas-update/environment-variables.md

```diff
@@ -80,9 +76,9 @@ API_URL="https://prod.example.com" eas update --branch production
 
 When EAS CLI creates the update, it will set the `API_URL` to `https://prod.example.com` inside the update's bundle.
 
-To access it, we can use the `expo-constants` library. It's located under the `Constants.manifest2?.extra?.expoClient?.extra?.eas?.API_URL` property.
+To access it, we can use the `expo-constants` library. It's located under the `Constants.expoConfig.API_URL` property.
 
-Note: we could also use the `expo-updates` library to access `API_URL`. It is under `Updates.manifest?.extra?.expoClient?.extra?.eas?.API_URL`. However, `Updates.manifest` is only present when an update is currently running. If the project is in development, `Updates.manifest` will be `undefined`. In addition, if a build is running without an update (e.g. it was just downloaded or there are no updates yet), `Updates.manifest` will also be `undefined`.
+> Note: we could also use the `expo-updates` library to access `API_URL`. It is under `Updates.manifest?.extra?.expoClient?.extra?.eas?.API_URL`. However, `Updates.manifest` is only present when an update is currently running. If the project is in development, `Updates.manifest` will be `undefined`. In addition, if a build is running without an update (e.g. it was just downloaded or there are no updates yet), `Updates.manifest` will also be `undefined`.
```

Comment:
```suggestion
> Note: We could also use the `expo-updates` library to access `API_URL`. It is under `Updates.manifest?.extra?.expoClient?.extra?.eas?.API_URL`. However, `Updates.manifest` is only present when an update is currently running. If the project is in development, `Updates.manifest` will be `undefined`. In addition, if a build is running without an update (for example, it was just downloaded or there are no updates yet), `Updates.manifest` will also be `undefined`.
```

Just a small suggestion to avoid latin abbreviation. Unrelated to the changes of this PR though.